packages/rocketchat-lg-api-extensions/package.js

```diff
@@ -12,6 +12,7 @@ Package.onUse(function (api) {
   api.use([
     'ecmascript',
     'deepwell:raven@0.3.0',
+    'learnersguild:rocketchat-lg-core@1.0.0',
```

Comment:
Here, it says `rocketchat-lg-core@1.0.0`, but above (in `.meteor/versions`) is says `rocketchat-lg-core@1.0.1`. Why come?